js/src/components/campaign-conversion-notice/migration-notice.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { Notice } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import localStorage from '.~/utils/localStorage';
+
+/**
+ * Shows Notice {@link Notice}
+ * It is the wrapper for the campaign convertion notices
+ *
+ *
+ * @param {Object} props React props.
+ * @param {JSX.Element} props.children Children to render.
+ * @param {string} props.className Class to use in the Notice component.
+ * @param {Object} props.localStorageKey Local Storage Key where is keep the dismiss state.
+ * @return {JSX.Element} {@link Notice} element with the info message and the link to the documentation.
+ */
+const MigrationCampaignNotice = ( {
+	children,
+	className,
+	localStorageKey,
+} ) => {
+	const defaultDismissedValue = !! localStorage.get( localStorageKey );
```

Comment:
I had the same feeling! I made some changes to the component, basically:
- I updated the name.
- It checks if the `localStorageKey ` is set or not.

However, I think if we want to leave it as an independent component and it would be convenient to add some callback function on the onRemove function so it is more customized depending on the user's needs. I think for now I will leave it as it is and in a different PR, we can focus to make this component a standard shared component.

Reply:
Makes sense, thanks for the explanation!